### PR TITLE
update volcano scheduler to 1.8.0; ignore vendor

### PR DESCRIPTION
**Pull request by lowang-bh**
Fix issue #586
To keep update for volcano release 1.8.0

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ profile.cov
 hack/python-sdk/openapi-generator-cli.jar
 dep-crds/
 dep-manifests/
+vendor/
```
Comment (alculquicondor) [inline .gitignore:9]: why is this necessary?

Comment (lowang-bh) [reply .gitignore:9]: In case of git add . to include the vendor dir.

Comment (tenzen-y) [reply .gitignore:9]: @lowang-bh We don't use vendoring in this repository. So I don't think we need this ignoring. Are there any reasons for this ignore?

Comment (lowang-bh) [reply .gitignore:9]: @tenzen-y sometimes user runs `go mod vendor`.  In case of `git add .` to include it. I can remove it if it is really not need.

Comment (tenzen-y) [reply .gitignore:9]: I'm ok with adding `vendor` to `.gitignore`.

**Review by alculquicondor — COMMENTED** [review]
/approve
